### PR TITLE
[issue/7] - added motivation part

### LINKED ISSUE #7: Write README description

### PR DESCRIPTION
**Pull request by AnnaGsell**
- Installation and Usage chapters are missing, because they should be added when more information is availabe (#30)

closes #7

### CODE REVIEW

**Review by diehmlukas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# sotired
```
Comment (mac641) [inline README.md:1]: I'd keep the heading and adjust it to the repository name

**Review by mac641 — CHANGES_REQUESTED** [review]
Just two minor things ;)

**Review by mac641 — DISMISSED** [review]
😄👍🏿

**Review by diehmlukas — DISMISSED** [review]

**Review by carina-str — COMMENTED** [review]
just a few grammatical improvments, as far as I can tell

```
soTired is an application for cognitive fatigue assessment.
It includes a stand-alone Android-App for fatigue detection and **an additional part** for data management and further analysis.
The project is structured in a client side (stand-alone App) implemented in Dart/Flutter and a server side (data management) in Golang.
Dart/Flutter **provides** the possibility to simply add an **iOS, Android** and / or Web-Application.
```

**Review by treilik — COMMENTED** [review]
please add a section for contributions, in which the code tags: "FIXME", "TODO" and "NOTE" are mentioned.

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# soTired
\ No newline at end of file
+# soTired
+
+## Motivation
+
+soTired is an application for cognitive fatigue assessment.
+It includes a stand-alone Android-App for fatigue detection and an additional part for data management and further analysis.
```
Comment (mac641) [inline README.md:6]:
```suggestion
It includes a stand-alone Android app for fatigue detection and an additional part for data management and further analysis.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
...
+The project is structured in a client side (stand-alone App) implemented in Dart/Flutter and a server side (data management) in Golang.
```
Comment (mac641) [inline README.md:7]:
```suggestion
The project is structured as client side (stand-alone app) implemented in Dart/Flutter and a server side (data management) written in Golang.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,12 @@
...
+Dart/Flutter provide the possibility to simply add an iOS and / or Web-Application.
```
Comment (mac641) [inline README.md:8]:
```suggestion
Dart/Flutter provides the ability to simply add an iOS, desktop and / or web application besides Android.
```

File: README.md [hunk 5]
```diff
@@ -1 +1,12 @@
...
+
+
+soTired is a remake of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
```
Comment (mac641) [inline README.md:11]:
```suggestion
soTired is a rewrite of the application presented in the [Validation of a Smartphone-Based Approach to In Situ Cognitive Fatigue Assessment](https://mhealth.jmir.org/2017/8/e125) paper by Edward Price, George Moore, Leo Galway and Mark Linden.
```

**Review by mac641 — CHANGES_REQUESTED** [review]
I've added some suggestions, you've previously introduced to me 😁

**Pull request closed without merging**